.vagrant/machines/default/virtualbox/private_key

```diff
@@ -0,0 +1,27 @@
+-----BEGIN RSA PRIVATE KEY-----
```

Comment:
Seems like a bad practice to include private keys in our repo.

Reply:
agreed, private keys should not be in repos many reasons.